tcms/issuetracker/types.py

```diff
@@ -343,16 +343,29 @@ class Redmine(IssueTrackerType):
     def is_adding_testcase_to_issue_disabled(self):
         (api_username, api_password) = self.rpc_credentials
 
-        return not (self.bug_system.base_url and api_username and api_password)
+        return not (
+            self.bug_system.base_url and (
+                self.bug_system.api_access_key or (
+                    self.bug_system.api_username and self.bug_system.api_password
+                )
+            )
+        )
```

Comment:
Just remove `api_username` from the original condition and it should be good enough.

Reply:
Done, updated as requested.